shared/teams/team/rows/emoji-row/item.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react'
+import * as Styles from '../../../../styles'
+import * as RPCChatTypes from '../../../../constants/types/rpc-chat-gen'
+import * as Kb from '../../../../common-adapters'
+import * as dateFns from 'date-fns'
+import EmojiMenu from './emoji-menu'
+type OwnProps = {
+  emoji: RPCChatTypes.Emoji
+}
+
+const ItemRow = ({emoji}: OwnProps) => {
+  let menuRef = React.useRef<Kb.Box>(null)
```

Comment:
`Kb.usePopup` does the thing you're doing on this and the next 6 lines, and it defers actually making the popup before it needs to be shown.